docs/running-on-kubernetes.md

```diff
@@ -0,0 +1,206 @@
+---
+layout: global
+title: Running Spark on Kubernetes
+---
+
+Support for running on [Kubernetes](https://kubernetes.io/) is available in experimental status. The feature set is
+currently limited and not well-tested.
+
+## Setting Up Docker Images
+
+In order to run Spark on Kubernetes, a Docker image must be built and available on the Docker registry. Spark
```

Comment:
"and available in an accessible Docker registry" (thinking about on-prem, internet-isolated use cases)